gulp/angular/dev-app/angulardevapp.gulpfile.js

```diff
@@ -0,0 +1,70 @@
+const gulp = require('gulp');
+const exec = require('child_process').exec;
+const spawn = require('child_process').spawn;
+const clean = require('gulp-clean');
+
+gulp.task('build-angular-dev-app', (cb) => {
```

Comment:
i can change them to ng

Reply:
actually, i think i'd rather keep angular, because some of the time the command refers to the directory and i dont want to change its name. I want to keep the use of angular / ng consistant.